### ISSUE TITLE
Adding masks to the upper forehead/lower chin

### ISSUE THREAD

**Issue by codyreading** [wpeebles/gangealing#31]
Hello!

I was wondering if it was possible to add masks on the upper forehead/lower chin. The congealed images are are cut off at both the forehead and the chin (See [here](https://github.com/wpeebles/gangealing/blob/main/assets/averages/celeba.png)). I was hoping to apply a mask on the full face including these regions.

Thanks and very cool work!

**Comment by wpeebles** [comment]
Hi @codyreading. If you fine-tune our CelebA model with `--ndirs 1` argument, you would probably get the full-face template you're looking for. Alternatively, I guess it may also be possible to extrapolate the flow to those regions, but that's not natively supported in the code unfortunately :/

**Comment by codyreading** [comment]
Hi @wpeebles ,

Perfect, will give that a try. Are there any other default arguments I need for CelebA? I noticed there wasnt a training script for CelebA in https://github.com/wpeebles/gangealing/tree/main/scripts/training

**Comment by wpeebles** [comment]
Oops, sorry about that. Just pushed a base training script for In-The-Wild CelebA. You'll want to remove the `--load_G_only` argument. Since you're training with a different value of `ndirs` than the base model, you'll also want to use lines 230-235 of `train.py` to initialize `ll` instead of resuming it and its optimizer from the state dict. In addition to decreasing `ndirs`, you may also need to decrease `--inject` to 5 or so in order to get the template you want.

Btw, I should mention I haven't tried fine-tuning our STNs before, but it seems worth trying before training a new model from scratch. The good news though is that CelebA is the fastest model to train/fine-tune since the StyleGAN has a resolution of 128x128

**Comment by codyreading** [comment]
Yup that makes sense, will give this a try thanks!
